src/apache/utilities/apache-utilities

```diff
@@ -153,3 +154,51 @@ _apache_set_previous_http_compression()
 {
 	snapctl set private.http.compression="$1"
 }
+
+apache_notify_push_enabled()
+{
+	[ "$(apache_get_notify_push_enabled)" = "true" ]
+}
+
+apache_enable_notify_push_enabled()
+{
+	_apache_set_notify_push_enabled "true"
+}
+
+apache_disable_notify_push_enabled()
+{
+	_apache_set_notify_push_enabled "false"
+}
+
+apache_notify_push_enabled_has_changed()
+{
+	[ "$(apache_get_notify_push_enabled)" != "$(_apache_get_previous_notify_push_enabled)" ]
+}
+
+apache_get_notify_push_enabled()
+{
+	compression="$(snapctl get http.notify-push)"
```

Comment:
`compression`?

Reply:
Is that a local variable?

Reply:
I renamend it to `notify_push` in that method, looks like it's not used somewhere else.